stream-compositions/services/legal-entity-composition-service/src/main/java/com/backbase/stream/compositions/legalentity/core/config/LegalEntityConfiguration.java

```diff
@@ -0,0 +1,44 @@
+package com.backbase.stream.compositions.legalentity.core.config;
+
+import com.backbase.stream.compositions.integration.legalentity.ApiClient;
+import com.backbase.stream.compositions.integration.legalentity.api.LegalEntityIntegrationApi;
+import com.backbase.stream.compositions.legalentity.core.mapper.LegalEntityMapper;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.AllArgsConstructor;
+import org.mapstruct.factory.Mappers;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.web.reactive.function.client.WebClient;
+
+import java.text.DateFormat;
+
+@Configuration
+@AllArgsConstructor
+@EnableConfigurationProperties(LegalEntityConfigurationProperties.class)
+public class LegalEntityConfiguration {
+    private final LegalEntityConfigurationProperties legalEntityConfigurationProperties;
+
+    @Bean
+    public LegalEntityMapper mapper() {
```

Comment:
Should use mapstructs `componentModel` (https://mapstruct.org/documentation/stable/reference/html/#using-dependency-injection)

Reply:
any update on this @BB-piotr ?